### PR TITLE
Update uswds using `uswds-gulp`

### PR DESCRIPTION
**Pull request by lauraGgit**
## Description

work on migrating tock to uswds 2.13.0 via converting the assets from vendored to pipeline-driven - by leveraging the gulpfile configuration that uswds provides.

This approach includes the autoprefixer.

## This pr includes

- [x] configuration for uswds
- [x] removing uswds v.2.3 (most of the file changes)
- [x] some additional css tweak
- [x] one massive package-lock.json
- [x] update to base.html loading of `uswds.min.js` so that it will avoid a race condition.
- [x] update docs to include uswds updates
- [x] change `sass` docker container name to `gulp` to reflect new container function
- [x] this should also address the squares in #1403

### CODE REVIEW

**Review by neilmb — APPROVED** [review]
This seems like it was probably hard-won to succeed with upgrading USWDS. Do we want to add any notes in `docs/` about how to accomplish the USWDS version upgrade?

**Comment by lauraGgit** [comment]
@neilmb if you have a change to give another review prior to us merging that would be great.

File: docs/dependency-management.md [hunk 1]
```diff
@@ -43,14 +43,16 @@ The Python version used in production is specified in `runtime.txt`. When updati
 3. Update the `python_version` line in `Pipfile` to ensure dependencies for the correct Python version are used
 4. Update `.circleci/config.yml` to ensure the correct Python version is used in CI
 
+## USWDS
+The uswds is initially installed via `npm` and then is compiled via gulp using the [uswds-gulp](https://github.com/uswds/uswds) configuration. To update USWDS, update the 2.x version in the `package.json` and review the release notes to identify any breaking changes.
```
Comment (neilmb) [inline docs/dependency-management.md:47]: Thanks for including this here.

File: tock/tock/static/sass/core.scss [hunk 2]
```diff
@@ -17,7 +15,7 @@ $theme-font-path:          "../../vendor/uswds_v2.3.1/fonts";
   position: absolute; // for some reason, USWDS 2.x overrides this to set position as relative, which breaks nav
 }
 
-
+//test comment --lg 11
```
Comment (neilmb) [inline tock/tock/static/sass/core.scss:18]: **question:** Is this actually used in a test case or should it be taken out to keep things clean?

**Review by neilmb — APPROVED** [review]

**Pull request merged**